README.md

```diff
@@ -278,4 +271,12 @@ This behavior is also seen in resource types from the following RPs:
 - Microsoft.DBforMySQL
 - Microsoft.HDInsight
 
+### Azure Policy Extension for Arc is not compatible on Kubernetes 1.25 (preview) version
+
+Policy extension for Arc installation will fail on 1.25 clusters with the following error code via gatekeeper controller and policy webhook pods: 
```

Comment:
consider removing "via gatekeeper controller and policy webhook pods"

Reply:
fixed!